airflow/contrib/hooks/bigquery_hook.py

```diff
@@ -194,6 +194,7 @@ def run_query(
             udf_config = False,
             use_legacy_sql=True,
             maximum_billing_tier=None,
+            schema_update_options=(),
```

Comment:
for same backwards-compatibility reason, please move `schema_update_options` as the last argument for this method.